src/diffusers/loaders/unet.py

```diff
@@ -15,7 +15,7 @@
 from collections import defaultdict
 from contextlib import nullcontext
 from pathlib import Path
-from typing import Callable, Dict, Union
+from typing import Callable, Dict, Optional, Union
```

Comment:
Think the changes in this module can safely be ignored. `load_attn_procs` has long been deprecated.

Reply:
Safely.

Reply:
Reverted.